examples/application_profile/mod_spectro_org.tf

```diff
@@ -0,0 +1,85 @@
+data "spectrocloud_registry" "common_registry" {
+  name = "Public Repo"
+}
+
+data "spectrocloud_registry" "container_registry" {
+  name = "vishwa-pack-registry"
+}
+
+data "spectrocloud_registry" "db_registry" {
+  name = "svtest"
+}
+
+data "spectrocloud_registry" "bitnami_registry" {
+  name = "Bitnami"
+
+}
+
+data "spectrocloud_pack_simple" "redis_pack" {
+  type         = "operator-instance"
+  name         = "redis-operator"
+  version      = "6.2.12-1"
+  registry_uid = data.spectrocloud_registry.common_registry.id
+}
+
+data "spectrocloud_pack_simple" "mysql_pack" {
```

Comment:
as this module is designed to reduce need in tf code please rewrite example without data source.

Reply:
Removed data source from examples